.github/workflows/renovate.yml

```diff
@@ -0,0 +1,26 @@
+name: Renovate
+on:
+  push:
+    branches:
+      - main
+    paths-ignore:
+      - ".github/ISSUE_TEMPLATE/*"
+      - "**.md"
+  schedule:
+    # Quotes required for "*" (#42, asterisk), special semantics in YAML
+    - cron: "0/15 * * * *"
```

Comment:
Is this running every 15 minutes? It seems kind of a lot. It can be executed once a day `"0 0 * * *"`

Reply:
Whoops! Totally correct. This was from Mend's sample.